bindings/nodejs/lib/index.d.ts

```diff
@@ -99,7 +99,7 @@ export declare class Client {
   getAddressBalance(address: string): Promise<AddressBalance>
   isAddressValid(address: string): boolean
   getMilestone(index: number): Promise<MilestoneMetadata>
-  getMilestoneUTXOChanges(index: number): Promise<MilestoneUTXOChanges>
+  GetMilestoneUtxoChanges(index: number): Promise<MilestoneUTXOChanges>
```

Comment:
A popular convention in Javascript is to only capitalize constructors, for methods the first world should be lowercase, i.e., `getMilestoneUtxoChanges`

Reply:
Yeah, but Clippy complained about it, should I add some ignore rule?

Reply:
The Clippy complained about `UTXO`, which should be `Utxo`. the `get` prefix should be OK.

Reply:
Oh right, sry read it wrong

Reply:
No no don't be. This PR is quite good.